addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3526,6 +3526,7 @@ const SnippetOptionWidget = Widget.extend({
     _renderXMLWidgets: function (parentEl, parentWidget) {
         const proms = [...parentEl.children].map(el => {
             const widgetName = el.tagName.toLowerCase();
+            // eslint-disable-next-line no-prototype-builtins
```

Comment:
No, let's not add this kind of stuff while everybody has a different linter x) Just disable that warning in your config.